flutter-studio/src/io/flutter/project/FlutterProjectStep.java

```diff
@@ -209,6 +220,20 @@ public static void ensureComboModelContainsCurrentItem(@NotNull final JComboBox
     comboBox.getEditor().setItem(currentItem); // to set current item in combo itself
   }
 
+  private void updateSdkField(JTextComponent sdkEditor) {
+    FlutterSdk current = FlutterSdk.forPath(sdkEditor.getText());
+    Color color = sdkBackgroundColor;
+    if (current == null) {
+      if (ColorUtil.isDark(sdkBackgroundColor)) {
+        color = ColorUtil.darker(JBColor.YELLOW, 5);
```

Comment:
It seems to me that there may be some way of getting an 'error' or 'invalid' background color from the current theme, but some very brief looking didn't turn up anything.

Reply:
Yeah, I found a bunch of named colors, but nothing useful. We can keep looking.